Class-8-(Intro-to-Spring-and-Controllers).md

```diff
@@ -0,0 +1,34 @@
+# Class 8: Introduction to Spring Boot, MVC, and Controllers
+
+## In the prep work this week, the students learned
+
+1. About the MVC design pattern.
+1. What the Spring framework is.
+1. How to initialize a Spring Boot project.
+1. How to handle routing and write controllers in Spring
+
+## Announcements
+
+1. Check with your course manager for any important announcements.
+
+## Large Group Time (Instructor)
+
+### Class 8 Topics That Require Careful Attention
+
+1. The Java/Gradle development environment can be tricky. Some students may encounter that we were not able to anticiapte in the text. Some minor versions may be different than what is listed in the 
```

Comment:
Is there a missing word in the sentence, "Some students may encounter..."?  Is it "problems"?

Also, I'm not sure what you mean by "Some minor versions".  Are you referring to different local Java/Gradle/Spring versions vs. those we use in the book?